Integrations/integration-sampleSiem.yml

```diff
@@ -0,0 +1,134 @@
fXjfVWbqPamqhUbsuBdd3FCLldNkkom1Rmg3QSlly2apruzVOj3JnJS4NaAAAAAALxPZT/AFbdUvLhbXiGzMnM70aloWqjMkAAAAAAAAzWN/3rb9Wf8CPsz4aN9nhR4+qs5oczqgAAAAADXcwP+w7DPJRjvxQo55k5zrXerkAAAAAAACu72lnTapmDQZTsyUWmouro0yZGpF3OnfC4nfSY2vuCdjXrT2DdKbuSnGuB7b1ScTzyxik0pSk0IwhxuHvy9WFeHslEs+42qgAAAAAAAAAAAAAAAAAAAAAAAAhGMsYTSxjCMIwjCMI8EYRh8sIwjD5YRhEC7nsM7zr/ADApQdE2re8X1UyxCRVtgjMV1VGd47yW0bISqS4vvWrOZJnK2QWDZFVWkVN2tPGvISxaLTy1JNt9aZL1rD8+nL2rRK2AZkgAAAAAAAAAAAAAAAABTl7Wf/IF/VT/AG4Grlvi8PrVlTmNSAAAAAANE/s33Vf2Lyr5f8ZoGG/xFoyTyHFIAAAAAAAAAAAAACOLd56s3WjyH3J88xOlriRrROTLpPQVAAAAAAn87NX1mVH5D8s/M0E48xw/FMZtDcwrAAAAAAAAAAAAqq9qw9W3S1y4XL4hvDTy29OpEqOxrVAAAAAAvE9lP9W3VLy4W14hszJzO9GpaFqozJAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMdio/lB9+GOfnpz1FHwwAAAAA7S23usK0M+11p09LlpFa9ydUkZtWE81cAAAAAAAAAAAEP2/d1S+rb8Dwx9ojEZ1s8WPH0ROTNNN6oAAAAAFl/ssfr6Zq9kW8vTLgw4cxuRr96YzX1DEsAAAAABnE9oj61TN/6HYV9E9pm+xw4VnNCEdUAAAAAAAAAAAAAAAAAAAAavO356heiT2RdNvoasw86vfnXK8ZOuigAAAAAAAAAAGOYeooAAAAAB3vtZdY9oj9pbE/jZTylzhzqIzag2ScdWdl3Ht74syHRULjsTItq16y7voTmdZJKq27clNc0mrMvpDZRF01UWZOp4SLIzyLIz8E6c0s8ss0PPiZicYzXZYOvDSDeuhjVHlDTneUXb9C06vF/Yt1uGM7FC/ca1uM72yryaSfftYqVKlTQRfpN1XCLGrN3bPus87aeJ6NFUV07UKOQCwAAAAABcM7Mbr7mQe3Vt85Fq8Iov8A84Mo6dnL50tMpK9RRjU8n4yYyru1E4IrMm6tz09u3QTlkmRrayyk0yqMsMvMUfHHimJ9i5OZVgDjvXrrAs3QvpYyjqKu76I9eWvSJqbYFruXEqE98ZNrkirKx7SShKsk6mavqtGC9RUbwUWZ0hs7dwkng3mljaima6opgZYGQ7/u7K1+3pk6/wCtOLjvnId1V+9bwr7pNui4rNy3PVHVZrdSUQaJN2jeLyovFJ4JIppopwjxZJZZYQlh6MRERhGSj8eSAAAAAAXT+zMbfUlu2tcW4Bk6gyfXt4pVewNOreoIN1FKXaLZytTMh5IaJqyLqNntzVRtPQqevLFu4SYs6lCMFG1QTmjk5ivt2I8UxHtdI9qY6vzD3tjY+9CmoQjl9+dXuTOSg+bFQAAAAALxPZT/AFbdUvLhbXiGzMnM70aloWqjMkAAAAAAAAzWN/3rb9Wf8CPsz4aN9nhR4+qs5oczqgAAAAADXcwP+w7DPJRjvxQo55k5zrXerkAAAAAAAD8jkCxbXyjYd7YzvimSVuysiWjcli3hRlFl26dXte7qM9t+4KZO4aqIukJH9JqCyUZ055FJYT8MsYRhCJMTMTjGYyptbek++dE2pzKunO+0Xarix7gXjatwuG6KCF74+qsY1Cxr2YwauHbOCVxW8siquimsrFg+guzVjK4brSS+jRVFdMVQo5TLAAAAAAAAAAAAAAAAAAAAAAAAAfY0esVa3qtS6/Qam/otcodRY1ii1mlPHFPqlJq1McpPadU6a/aKJOmL9g8QkVRWSmlUSUkhNLGEYQiBpf7OW4U33B9J1Iui6HjOXPGJ12GPc5U1CLRFSoV1Jh3W38itqe1glBnScj0ttO54JUUEEqq2qDZCWKTaWabBdo2KuzdnJaJxSxnJIAAAAAAAAAAAAAAAApy9rP8A5Av6qf7cDVy3xeH1qypzGpAAAAAAGif2b7qv7F5V8v8AjNAw3+ItGSeQ4pAAAAAAAAAAAAAARxbvPVm60eQ+5PnmJ0tcSNaJyZdJ6CoAAAAAE/nZq+syo/Ifln5mgnHmOH4pjNobmFYAAAAAAAAAAAFVXtWHq26WuXC5fEN4aeW3p1IlR2NaoAAAAAF4nsp/q26peXC2vENmZOZ3o1LQtVGZIAAAAAAAAAAAAAAAAAAAAAAAAAAAABjsVH8oPvwxz89Oeoo+GAAAAAHaW291hWhn2utOnpctIrXuTqkjNqwnmrgAAAAAAAAAAAh+37uqX1bfgeGPtEYjOtnix4+iJyZppvVAAAAAAsv9lj9fTNXsi3l6ZcGHDmNyNfvTGa+oYlgAAAAAM4ntEfWqZv8A0Owr6J7TN9jhwrOaEI6oAAAAAAAAAAAAAAAAAAAA1edvz1C9Ensi6bfQ1Zh51e/OuV4yddFAAAAAAAAAAAMcw9RQAAAAADvfay6x7RH7S2J/GynlLnDnURm1PDzl1d3tEe35Jqe0vw1K49osrnNelulVWvVGRkgzlfXng2bjVG/KG4VmkRcvHdhRSmuKnSzrzSot06qgggq5fycHexXs1bM5T6omGfcbVQAAAAAP32KsnXrhXJViZcxxWlrevzG110O87TrKEsqkWNct+oIVJhOs3UhMg9ZqLIQkcN1YTIuUJp0lJZpJ5pYxMRMYTkNUfRFqwsrW1phxVqOsiCLJvfVBlhc1tyO5Hbmy78o6s9KvS0Hs33i/DRa+1WlbKqpozvGE7d3LJKm4k4fOrpmiqaZXdXFRn69or1+R1Lao5NM+P61M6w3pZqNUoNWUZLuYMLuzmvLBjflVXSmg3Tco4/glG3WfdEpoou
```

Comment:
I think that we need to remove this whole phase thing from the incident creation
@rishibhargava @slavikm please approve